modules/nw-dpu-running-workloads.adoc

```diff
@@ -16,7 +16,7 @@ Follow these steps to deploy a workload on the DPU.
 
 .Procedure
 
-. Create a sample workload on the host side by using the following YAML, save the file as `workload-host.yaml`:
+. Create a sample workload designed to run on the host-side worker node by using the following YAML, save the file as `workload-host.yaml`:
```

Comment:
There should be a full stop after YAML. Then Save with a capital S. 
It's two separate sentences.